docs/tutorials/web.md

```diff
@@ -5,10 +5,400 @@ has_children: false
 nav_order: 7
 ---
 
-# Deploy ML models on the web
 
- Check out the [release blog here](https://cloudblogs.microsoft.com/opensource/2021/09/02/onnx-runtime-web-running-your-machine-learning-model-in-browser/)
+# Deploy and inference ML models on the web
+{: .no_toc }
 
+In this tutorial we will use a template QuickStart to start building out a static web app using ONNX Runtime web. We will do the inference in JavaScript on the browser for a computer vision model.
 
- *[DOCUMENTATION COMING SOON]*
+One of the hardest parts when deploying and inferencing in languages that are not commonly used for data science is figuring out how to do the data processing and inferencing. We have done all the hard work for you with [this template](https://github.com/microsoft/onnxruntime-nextjs-template)! 
 
+Below is what the site looks like from the template. It is looping through a list of example images, calling the inference session with a SqueezeNet model, then returning the score and label from inference.
+
+Example template output:
+
+<img src="../../images/ort-web-demo.gif" width="75%" height="75%"/>
```

Comment:
Can we make this a bit bigger? It's really hard to see the output. Also we should add alt text to the image for accessibility.

Reply:
Changed to 100%, is that better? Or still too small?

Reply:
Looks exactly the same to me. Don't worry about it for now. We can try and finesse it later

Reply:
Can we add the alt text though?

Reply:
good catch, added!